### PR TITLE
182275516 – updated documentation while  creating ngss-staging

### PR DESCRIPTION
**Pull request by knowuh**
All the scripts run well. I encountered a few errors while trying to create these stacks, but they were the result of user-error. To help smooth the way for the next time that we do this, I have updated the documentation here slightly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,33 +23,82 @@ This script should work for other types of stacks, but it hasn't been tested.
...
+        - SiteURL
+        - Environment
+        - DbHost
+        - DatabaseSecurityGroupId
+        - AuthoringSiteURL
+        - S3SecreateAccessKey
+        - S3AccessKeyId
+        - ReportDomainName
+        - LogStashDbHost
+        - ClusterStackName
```
Comment (scytacki) [inline README.md:72]: The numbering in messed up in markdown after this section. I'm guessing it has to do with the indentation.

File: scripts/package.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
   "author": "Scott Cytacki",
   "license": "MIT",
   "dependencies": {
-    "aws-sdk": "^2.328.0",
+    "aws-sdk": "^2.1147.0",
```
Comment (scytacki) [inline scripts/package.json:14]: Wow a 4 digit second segment. Crazy.

**Review by scytacki — CHANGES_REQUESTED** [review]
Looks good, just one issue I found with the markdown formatting.

File: README.md (hunk 1 above)
Comment (knowuh) [reply README.md]: I ran a Markdown Linter on this file and fixed several formatting issues. It turns out that the markdown for this section was valid, but the Github markdown engine seems to outdent unordered lists a bit when they are inside an ordered list. 

So I made the PARAMs list ordered even though they are non-sequential. I think it looks a bit better now.  Let me know what you think.

**Comment by knowuh** [comment]
@scytacki -- Could you review again please?

**Review by scytacki — APPROVED** [review]
looks good to me

**Pull request merged**